lib/workers/repository/updates/branchify.js

```diff
@@ -23,12 +25,10 @@ function branchifyUpgrades(config) {
         logger.debug(
           `Dependency ${upgrade.depName} is part of group ${upgrade.groupName}`
         );
-        upgrade.groupSlug =
-          upgrade.groupSlug ||
-          upgrade.groupName
-            .toString()
-            .toLowerCase()
-            .replace(/[^a-z0-9+]+/g, '-');
+        upgrade.groupSlug = slugify(upgrade.groupSlug || upgrade.groupName, {
+          remove: /^-|\.\.|\/$/g,
```

Comment:
This pattern needs to be explained/documented

Reply:
Removing this actually, letting this be handled via `clean-git-ref`